### PR TITLE
Implemented insurances sorting and paging

### PR DESCRIPTION
**Pull request by alex-bel**

### CODE REVIEW

File: src/main/java/com/hedvig/backoffice/services/product_pricing/ProductPricingServiceStub.java [hunk 1]
```diff
@@ -127,19 +136,65 @@ public void activate(String memberId, InsuranceActivateDTO dto, String token) {
...
+    put(ProductSortColumns.ACTIVE_TO_DATE, Comparator.comparing((InsuranceStatusDTO ins) -> ins.getInsuranceActiveTo(), nullsFirst(LocalDateTime::compareTo)).reversed());
+    put(ProductSortColumns.CANCELLATION_EMAIL_SENT_DATE, Comparator.comparing((InsuranceStatusDTO ins) -> ins.isCancellationEmailSent(), nullsFirst(Boolean::compareTo)).reversed());
+    put(ProductSortColumns.CERTIFICATE_UPLOADED, Comparator.comparing((InsuranceStatusDTO ins) -> ins.isCertificateUploaded(), nullsFirst(Boolean::compareTo)).reversed());
+    put(ProductSortColumns.CONTRACT_SIGNED_DATE, Comparator.comparing((InsuranceStatusDTO ins) -> ins.getSignedOn(), nullsFirst(Instant::compareTo)).reversed());
+    put(ProductSortColumns.HOUSEHOLD_SIZE, Comparator.comparing((InsuranceStatusDTO ins) -> ins.getPersonsInHouseHold(), nullsFirst(Integer::compareTo)).reversed());
+    put(ProductSortColumns.MEMBER_FULL_NAME, Comparator.comparing((InsuranceStatusDTO ins) -> ins.getMemberLastName() + " " + ins.getMemberFirstName(), nullsFirst(String::compareTo)).reversed());
```
Comment (meletios) [inline src/main/java/com/hedvig/backoffice/services/product_pricing/ProductPricingServiceStub.java]: FirstName first, Lastname last :smile:

File: src/main/java/com/hedvig/backoffice/services/product_pricing/ProductPricingServiceStub.java [hunk 2]
```diff
@@ -127,19 +136,65 @@ public void activate(String memberId, InsuranceActivateDTO dto, String token) {
...
+    put(ProductSortColumns.ACTIVE_FROM_DATE, Comparator.comparing((InsuranceStatusDTO ins) -> ins.getInsuranceActiveFrom(), nullsLast(LocalDateTime::compareTo)));
+    put(ProductSortColumns.ACTIVE_TO_DATE, Comparator.comparing((InsuranceStatusDTO ins) -> ins.getInsuranceActiveTo(), nullsLast(LocalDateTime::compareTo)));
+    put(ProductSortColumns.CANCELLATION_EMAIL_SENT_DATE, Comparator.comparing((InsuranceStatusDTO ins) -> ins.isCancellationEmailSent(), nullsLast(Boolean::compareTo)));
+    put(ProductSortColumns.CERTIFICATE_UPLOADED, Comparator.comparing((InsuranceStatusDTO ins) -> ins.isCertificateUploaded(), nullsLast(Boolean::compareTo)));
+    put(ProductSortColumns.CONTRACT_SIGNED_DATE, Comparator.comparing((InsuranceStatusDTO ins) -> ins.getSignedOn(), nullsLast(Instant::compareTo)));
+    put(ProductSortColumns.HOUSEHOLD_SIZE, Comparator.comparing((InsuranceStatusDTO ins) -> ins.getPersonsInHouseHold(), nullsLast(Integer::compareTo)));
+    put(ProductSortColumns.MEMBER_FULL_NAME, Comparator.comparing((InsuranceStatusDTO ins) -> ins.getMemberLastName() + " " + ins.getMemberFirstName(), nullsLast(String::compareTo)));
```
Comment (meletios) [inline src/main/java/com/hedvig/backoffice/services/product_pricing/ProductPricingServiceStub.java]: FirstName first, Lastname last :smile:

File: src/main/java/com/hedvig/backoffice/services/product_pricing/ProductPricingServiceStub.java (hunk 1 above)
Comment (alex-bel) [reply src/main/java/com/hedvig/backoffice/services/product_pricing/ProductPricingServiceStub.java]: Done

File: src/main/java/com/hedvig/backoffice/services/product_pricing/ProductPricingServiceStub.java (hunk 2 above)
Comment (alex-bel) [reply src/main/java/com/hedvig/backoffice/services/product_pricing/ProductPricingServiceStub.java]: Done

**Pull request merged**